src/coreclr/vm/genericdict.cpp

```diff
@@ -1041,28 +1041,8 @@ Dictionary::PopulateEntry(
 
             if (fRequiresDispatchStub)
             {
-                // Generate a dispatch stub and store it in the dictionary.
```

Comment:
I think it might make sense to keep some of this comment here even if the rest of it moves to GenerateDispatchStubCellEntrySlot. i.e. the "why are we doing this" might belong here next to `fRequiresDispatchStub` and the "how are we doing this" can live in GenerateDispatchStubCellEntrySlot